src/status_im/ui/screens/wallet/accounts/views.cljs

```diff
@@ -290,7 +291,7 @@
           [quo2.button/button
            {:type     :grey
             :size     32
-            :on-press #(re-frame/dispatch [:bottom-sheet/show-sheet
+            :on-press #(re-frame/dispatch [:bottom-sheet/show-sheet-old
```

Comment:
Why all these "old" sheets?

Reply:
because we still have old sheets in old UI, and they implemented in different designs so we have to keep old bottom sheet for now